### PR TITLE
Add simple example testcafe tests

### PR DESCRIPTION
**Pull request by neilschark**

### CODE REVIEW

**Review by andresterba — CHANGES_REQUESTED** [review]
Good job getting this is an working state!

File: app/src/components/user-page/user-page.vue [hunk 1]
```diff
@@ -36,27 +38,20 @@
...
                     </v-chip>
                 </template>
             </v-data-table>
         </v-card>
 
-        <change-user-names-modal
-            :user="editedUser"
-            v-model="showNamesDialog"
-        />
+        <change-user-names-modal :user="editedUser" v-model="showNamesDialog" />
```
[the remark this answers is not in the document]
Comment (neilschark) [reply app/src/components/user-page/user-page.vue:52]: eslint

File: app/src/components/user-page/user-page.vue [hunk 2]
```diff
@@ -36,27 +38,20 @@
...
 
-        <change-user-password-modal
-            v-model="showPasswordDialog"
-        />
+        <change-user-password-modal v-model="showPasswordDialog" />
```
[the remark this answers is not in the document]
Comment (neilschark) [reply app/src/components/user-page/user-page.vue:54]: eslint

File: testcafe/tests/admin-page-test.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    await t
+    .typeText('#input-18', 'Admin')
+    .typeText('#input-21', 'Admin')
+    .click("#login-loginbutton")
+});
+
+fixture `Admin View Navigation`
+    .page `http://127.0.0.1:8080/`;
+
+test('Navigation Tabs', async t => {
```
[the remark this answers is not in the document]
Comment (neilschark) [reply testcafe/tests/admin-page-test.js:13]: I have no idea if its possible.

[the remark this answers is not in the document]
Comment (neilschark) [reply testcafe/tests/admin-page-test.js:13]: But its expressive. It tests if you can navigate through the admin view: "Admin View Navigation"

**Pull request merged**